utils/questions.py

```diff
@@ -1,10 +1,286 @@
 from typing import Any
 
 import requests
+import random
 
 from bot_globals import logger
 from embeds.misc_embeds import error_embed
+from embeds.questions_embeds import question_embed
+from utils.run_blocking import to_thread
+from utils.ratings import get_rating_data
+import re
+import ast
 
+@to_thread
+def get_random_question(difficulty):
```

Comment:
add typing to all functions. Add typing to show what type the parameters are, and what the function returns.